### PR TITLE
Minor edit

### PR DESCRIPTION
**Pull request by dcoeurjo**
Hop

### CODE REVIEW

**Comment by dcoeurjo** [comment]
Here you have my issues with Xcode.. weird stuff..

line 222, testVoronoiMap, the IDE wants me to add extra `{}` around `false, false` and later complains with the VoronoiMap constructor (not viable constructor).

**Comment by dcoeurjo** [comment]
BTW, you never use `isPeriodic()`

Compared to the previous version, there is a slight overhead due to the `if (isPeriodic[dim])` tests if I'm right.

File: src/DGtal/geometry/volumes/distance/VoronoiMap.h [hunk 1]
```diff
@@ -229,6 +229,11 @@ namespace DGtal
      */
     ~VoronoiMap();
 
+    /**
+     * Disabling default constructor.
+     */
+     VoronoiMap() = delete;
```
Comment (rolanddenis) [inline src/DGtal/geometry/volumes/distance/VoronoiMap.h:235]: By the way, C++ doesn't implicitly declare the default constructor if any other constructor is defined. So deleting it is not mandatory even if it is clearer.

**Comment by rolanddenis** [comment]
About the error with Xcode, I think it should accept the brace elision but since it is a common pitfall, I propose to add the extra `{}`:

``` C++
 bool testVoronoiMap( std::array<bool, 2> const& periodicity = { { false, false } } )
```

Yes, I designed `isPeriodic()` for user-interface only but I should used it instead of testing `myPeriodicitySpec` directly.

I'm agree that the only overhead is on the periodicity tests since I specifically split the loops in generic and periodic parts. It should be negligible, isn't it ?

**Comment by dcoeurjo** [comment]
Yeah it's negligible. 
An alternative would have been to consider a static periodicity map (in template parameters) allowing the compiler to optimize the code (and cut branches) at compile time but it  would probably have been less interesting for the user.

**Comment by dcoeurjo** [comment]
BTW, it was not an error from Xcode, just a warning.. my bad

**Comment by rolanddenis** [comment]
Yes, I also think about it but since other DGtal classes don't work this way (actually only `KhalimskySpaceND` :wink: ), it would complicate user life. However, with additional work, it is possible to have a class that works both way (like `Eigen` with `Dynamic` magic value).

**Comment by rolanddenis** [comment]
Can I merge ?

**Comment by dcoeurjo** [comment]
Yep please. Try to merge the commits without closing my PR (and don't know if the PR will still be open with no diff...). 
I'll probably help you to polish the doc according to this new cool feature.

**Comment by dcoeurjo** [comment]
;) damit, it automatically closes the PR

**Comment by rolanddenis** [comment]
:smile:

**Comment by dcoeurjo** [comment]
ok I 've just pushed a tiny edit and i have to create a new PR so that you can see it...

**Comment by rolanddenis** [comment]
Ok, next time, I will directly merge from you branch ...

I will also push some documentation updates.

**Comment by dcoeurjo** [comment]
👌

**Pull request merged**